4-esm-webpack/package.json

```diff
@@ -2,23 +2,23 @@
   "name": "npm-es-modules-4-esm-webpack",
   "version": "0.0.2",
   "description": "Breakdown of 7 different ways to use ES modules with NPM today.",
-  "main": "main.js",
-  "module": "module.jsm",
+  "main": "index.js",
+  "module": "main.js",
   "browser": "dist/browser.js",
   "repository": "transitive-bullshit/npm-es-modules",
   "author": "Travis Fischer <travis@automagical.ai>",
   "license": "MIT",
   "scripts": {
     "prebuild": "yarn run clean",
-    "build": "webpack",
-    "start": "webpack -w",
+    "build": "webpack -r esm",
+    "start": "webpack -r esm -w",
     "clean": "del dist",
     "prepublishOnly": "yarn run build",
     "pretest": "yarn run build",
     "test": "ava -v && standard"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=6"
```

Comment:
`esm` works in Node 6+

Reply:
I had `>=8` because of the usage of `async await`. afaik esm doesn't transpile these types of features, correct?

Reply:
Ah yes, that's correct. If you use `await` you'll want Node 7.6+ _(so 8 being the lowest supported version by Node still)_.